### PR TITLE
docs: update EN FAQ

### PR DESCRIPTION
**Pull request by yzeng25**
### What this PR does / why we need it:

This PR will fix a description problem in our FAQ in English. 

- It helps explain how APISIX and APISIX Dashboard are related. Before this fix, the 'console interface' and 'web interface' are essentially describing a 'user interface' in different terms. So I decided to replace these phrases with 'user interface'. In this way, it is more straightforward. 

- According to the description in APISIX Dashboard, it is an experimental feature as of now. It is always good to remind potential users that this is an experimental feature.

- Replace 'APISIX' with 'Apache APISIX'

### Pre-submission checklist:

* [x] Did you explain what problem does this PR solve? Or what new features have been added?
* [x] Have you added corresponding test cases?
* [x] Have you modified the corresponding document?
* [x] Is this PR backward compatible? **If it is not backward compatible, please discuss on the [mailing list](https://github.com/apache/apisix/tree/master#community) first**

### CODE REVIEW

File: docs/en/latest/FAQ.md [hunk 1]
```diff
@@ -25,33 +25,33 @@ title: FAQ
...
+One of the goals of Apache APISIX design and development is the highest performance in the industry. Specific test data can be found here：[benchmark](benchmark.md)
 
-APISIX is the highest performance API gateway with a single-core QPS of 23,000, with an average delay of only 0.6 milliseconds.
+Apache APISIX is the highest performance API gateway with a single-core QPS of 23,000, with an average delay of only 0.6 milliseconds.
 
-## Does APISIX have a console interface?
+## Does Apache APISIX have a user interface？
 
-Yes, APISIX has a powerful Dashboard. APISIX and [APISIX Dashboard](https://github.com/apache/apisix-dashboard) are independent projects, you can deploy [APISIX Dashboard](https://github.com/apache/apisix-dashboard) to operate APISIX through the web interface.
+Yes. Apache APISIX has an experimental feature called [Apache APISIX Dashboard](https://github.com/apache/Apache APISIX-dashboard), which is an independent project apart from Apache APISIX. You can deploy Apache APISIX Dashboard to operate Apache APISIX through the user interface.
```
Comment (tao12345666333) [inline docs/en/latest/FAQ.md:44]:
```suggestion
Yes. Apache APISIX has an experimental feature called [Apache APISIX Dashboard](https://github.com/apache/apisix-dashboard), which is an independent project apart from Apache APISIX. You can deploy Apache APISIX Dashboard to operate Apache APISIX through the user interface.
```

File: docs/en/latest/FAQ.md [hunk 2]
```diff
@@ -92,7 +92,7 @@ There are two different ways to do this：
 1. Use the `vars` field of route to do it.
 
 ```shell
-curl -i http://127.0.0.1:9080/apisix/admin/routes/1 -H 'X-API-KEY: edd1c9f034335f136f87ad84b625c8f1' -X PUT -d '
+curl -i http://127.0.0.1:9080/Apache APISIX/admin/routes/1 -H 'X-API-KEY: edd1c9f034335f136f87ad84b625c8f1' -X PUT -d '
```
Comment (tao12345666333) [inline docs/en/latest/FAQ.md:95]:
```suggestion
curl -i http://127.0.0.1:9080/apisix/admin/routes/1 -H 'X-API-KEY: edd1c9f034335f136f87ad84b625c8f1' -X PUT -d '
```

File: docs/en/latest/FAQ.md [hunk 3]
```diff
@@ -105,7 +105,7 @@ curl -i http://127.0.0.1:9080/apisix/admin/routes/1 -H 'X-API-KEY: edd1c9f034335
     }
 }'
 
-curl -i http://127.0.0.1:9080/apisix/admin/routes/2 -H 'X-API-KEY: edd1c9f034335f136f87ad84b625c8f1' -X PUT -d '
+curl -i http://127.0.0.1:9080/Apache APISIX/admin/routes/2 -H 'X-API-KEY: edd1c9f034335f136f87ad84b625c8f1' -X PUT -d '
```
Comment (tao12345666333) [inline docs/en/latest/FAQ.md:108]:
```suggestion
curl -i http://127.0.0.1:9080/apisix/admin/routes/2 -H 'X-API-KEY: edd1c9f034335f136f87ad84b625c8f1' -X PUT -d '
```

File: docs/en/latest/FAQ.md [hunk 4]
```diff
@@ -135,7 +135,7 @@ There are several different ways to do this.
 1. Directly use the `http_to_https` in `redirect` plugin：
 
 ```shell
-curl http://127.0.0.1:9080/apisix/admin/routes/1  -H 'X-API-KEY: edd1c9f034335f136f87ad84b625c8f1' -X PUT -d '
+curl http://127.0.0.1:9080/Apache APISIX/admin/routes/1  -H 'X-API-KEY: edd1c9f034335f136f87ad84b625c8f1' -X PUT -d '
```
Comment (tao12345666333) [inline docs/en/latest/FAQ.md:138]: ditto

File: docs/en/latest/FAQ.md [hunk 5]
```diff
@@ -150,7 +150,7 @@ curl http://127.0.0.1:9080/apisix/admin/routes/1  -H 'X-API-KEY: edd1c9f034335f1
 2. Use with advanced routing rule `vars` with `redirect` plugin:
 
 ```shell
-curl -i http://127.0.0.1:9080/apisix/admin/routes/1  -H 'X-API-KEY: edd1c9f034335f136f87ad84b625c8f1' -X PUT -d '
+curl -i http://127.0.0.1:9080/Apache APISIX/admin/routes/1  -H 'X-API-KEY: edd1c9f034335f136f87ad84b625c8f1' -X PUT -d '
```
Comment (tao12345666333) [inline docs/en/latest/FAQ.md:153]: ditto

File: docs/en/latest/FAQ.md [hunk 6]
```diff
@@ -173,7 +173,7 @@ curl -i http://127.0.0.1:9080/apisix/admin/routes/1  -H 'X-API-KEY: edd1c9f03433
 3. `serverless` plugin：
 
 ```shell
-curl -i http://127.0.0.1:9080/apisix/admin/routes/1  -H 'X-API-KEY: edd1c9f034335f136f87ad84b625c8f1' -X PUT -d '
+curl -i http://127.0.0.1:9080/Apache APISIX/admin/routes/1  -H 'X-API-KEY: edd1c9f034335f136f87ad84b625c8f1' -X PUT -d '
```
Comment (tao12345666333) [inline docs/en/latest/FAQ.md:176]: ditto

File: docs/en/latest/FAQ.md [hunk 7]
```diff
@@ -200,7 +200,7 @@ Content-Type: text/html
 Content-Length: 166
 Connection: keep-alive
 Location: https://foo.com/hello
-Server: APISIX web server
+Server: Apache APISIX web server
```
Comment (tao12345666333) [inline docs/en/latest/FAQ.md:203]:
```suggestion
Server: APISIX web server
```

File: docs/en/latest/FAQ.md [hunk 8]
```diff
@@ -224,23 +224,23 @@ nginx_config:
...
+## How to make Apache APISIX listen on multiple ports when handling HTTP or HTTPS requests?
 
-By default, APISIX only listens on port 9080 when handling HTTP requests. If you want APISIX to listen on multiple ports, you need to modify the relevant parameters in the configuration file as follows:
+By default, Apache APISIX only listens on port 9080 when handling HTTP requests. If you want Apache APISIX to listen on multiple ports, you need to modify the relevant parameters in the configuration file as follows:
 
 1. Modify the parameter of HTTP port listen `node_listen` in `conf/config.yaml`, for example:
 
    ```
-    apisix:
+    Apache APISIX:
```
Comment (tao12345666333) [inline docs/en/latest/FAQ.md:243]: ditto

File: docs/en/latest/FAQ.md [hunk 9]
```diff
@@ -250,41 +250,41 @@ By default, APISIX only listens on port 9080 when handling HTTP requests. If you
    Handling HTTPS requests is similar, modify the parameter of HTTPS port listen `ssl.listen_port` in `conf/config.yaml`, for example:
 
     ```
-    apisix:
+    Apache APISIX:
```
Comment (tao12345666333) [inline docs/en/latest/FAQ.md:253]: ditto

File: docs/en/latest/FAQ.md [hunk 10]
```diff
@@ -250,41 +250,41 @@ By default, APISIX only listens on port 9080 when handling HTTP requests. If you
...
+* If the monitoring directory has data updates: etcd will return the new subscribed data immediately (in milliseconds), and Apache APISIX will update it to the memory cache.
 
-With the help of etcd which incremental notification feature is millisecond-level, APISIX achieve millisecond-level of configuration synchronization.
+With the help of etcd which incremental notification feature is millisecond-level, Apache APISIX achieve millisecond-level of configuration synchronization.
 
-## How to customize the APISIX instance id?
+## How to customize the Apache APISIX instance id?
 
-By default, APISIX will read the instance id from `conf/apisix.uid`. If it is not found, and no id is configured, APISIX will generate a `uuid` as the instance id.
+By default, Apache APISIX will read the instance id from `conf/Apache APISIX.uid`. If it is not found, and no id is configured, Apache APISIX will generate a `uuid` as the instance id.
```
Comment (tao12345666333) [inline docs/en/latest/FAQ.md:276]: ditto

File: docs/en/latest/FAQ.md [hunk 11]
```diff
@@ -250,41 +250,41 @@ By default, APISIX only listens on port 9080 when handling HTTP requests. If you
...
 
-If you want to specify a meaningful id to bind APISIX instance to your internal system, you can configure it in `conf/config.yaml`, for example:
+If you want to specify a meaningful id to bind Apache APISIX instance to your internal system, you can configure it in `conf/config.yaml`, for example:
 
     ```
-    apisix:
+    Apache APISIX:
```
Comment (tao12345666333) [inline docs/en/latest/FAQ.md:281]: ditto

File: docs/en/latest/FAQ.md [hunk 12]
```diff
@@ -335,26 +335,26 @@ Example: luarocks install luasec OPENSSL_DIR=/usr/local
...
+## How to access Apache APISIX Dashboard through Apache APISIX proxy
 
-1. Keep the APISIX proxy port and Admin API port different(or disable Admin API). For example, do the following configuration in `conf/config.yaml`.
+1. Keep the Apache APISIX proxy port and Admin API port different(or disable Admin API). For example, do the following configuration in `conf/config.yaml`.
 
 The Admin API use a separate port 9180:
 
 ```yaml
-apisix:
+Apache APISIX:
```
Comment (tao12345666333) [inline docs/en/latest/FAQ.md:345]: ditto

File: docs/en/latest/FAQ.md [hunk 13]
```diff
@@ -335,26 +335,26 @@ Example: luarocks install luasec OPENSSL_DIR=/usr/local
...
 
-2. Add proxy route of APISIX Dashboard:
+2. Add proxy route of Apache APISIX Dashboard:
 
-Note: The APISIX Dashboard service here is listening on `127.0.0.1:9000`.
+Note: The Apache APISIX Dashboard service here is listening on `127.0.0.1:9000`.
 
 ```shell
-curl -i http://127.0.0.1:9180/apisix/admin/routes/1  -H 'X-API-KEY: edd1c9f034335f136f87ad84b625c8f1' -X PUT -d '
+curl -i http://127.0.0.1:9180/Apache APISIX/admin/routes/1  -H 'X-API-KEY: edd1c9f034335f136f87ad84b625c8f1' -X PUT -d '
```
Comment (tao12345666333) [inline docs/en/latest/FAQ.md:354]: ditto

File: docs/en/latest/FAQ.md [hunk 14]
```diff
@@ -508,7 +508,7 @@ Blog https://juejin.cn/post/6965778290619449351 has detail setup.
 To strip route prefix before forwarding to upstream, for example from `/foo/get` to `/get`, could be achieved through plugin `proxy-rewrite`.
 
 ```shell
-curl -i http://127.0.0.1:9080/apisix/admin/routes/1 -H 'X-API-KEY: edd1c9f034335f136f87ad84b625c8f1' -X PUT -d '
+curl -i http://127.0.0.1:9080/Apache APISIX/admin/routes/1 -H 'X-API-KEY: edd1c9f034335f136f87ad84b625c8f1' -X PUT -d '
```
Comment (tao12345666333) [inline docs/en/latest/FAQ.md:511]: ditto

**Review by tao12345666333 — CHANGES_REQUESTED** [review]

File: docs/en/latest/FAQ.md (hunk 1 above)
Comment (yzeng25) [reply docs/en/latest/FAQ.md:44]: darn, my bad, will fix

File: docs/en/latest/FAQ.md [hunk 15]
```diff
@@ -335,26 +335,26 @@ Example: luarocks install luasec OPENSSL_DIR=/usr/local
...
-Note: The APISIX Dashboard service here is listening on `127.0.0.1:9000`.
+Note: The Apache APISIX Dashboard service here is listening on `127.0.0.1:9000`.
 
 ```shell
-curl -i http://127.0.0.1:9180/apisix/admin/routes/1  -H 'X-API-KEY: edd1c9f034335f136f87ad84b625c8f1' -X PUT -d '
+curl -i http://127.0.0.1:9180/Apache APISIX/admin/routes/1  -H 'X-API-KEY: edd1c9f034335f136f87ad84b625c8f1' -X PUT -d '
 {
     "uris":[ "/*" ],
-    "name":"apisix_proxy_dashboard",
+    "name":"Apache APISIX_proxy_dashboard",
```
Comment (tao12345666333) [inline docs/en/latest/FAQ.md:357]:
```suggestion
    "name":"apisix_proxy_dashboard",
```

File: docs/en/latest/FAQ.md [hunk 16]
```diff
@@ -476,26 +476,26 @@ The route was created successfully. It means that the modification of `X-API-KEY
...
 
 ```bash
-$ curl --output /root/.acme.sh/renew-hook-update-apisix.sh --silent https://gist.githubusercontent.com/anjia0532/9ebf8011322f43e3f5037bc2af3aeaa6/raw/65b359a4eed0ae990f9188c2afa22bacd8471652/renew-hook-update-apisix.sh
+$ curl --output /root/.acme.sh/renew-hook-update-Apache APISIX.sh --silent https://gist.githubusercontent.com/anjia0532/9ebf8011322f43e3f5037bc2af3aeaa6/raw/65b359a4eed0ae990f9188c2afa22bacd8471652/renew-hook-update-Apache APISIX.sh
 
-$ chmod +x /root/.acme.sh/renew-hook-update-apisix.sh
+$ chmod +x /root/.acme.sh/renew-hook-update-Apache APISIX.sh
 
-$ acme.sh  --issue  --staging  -d demo.domain --renew-hook "/root/.acme.sh/renew-hook-update-apisix.sh  -h http://apisix-admin:port -p /root/.acme.sh/demo.domain/demo.domain.cer -k /root/.acme.sh/demo.domain/demo.domain.key -a xxxxxxxxxxxxx"
+$ acme.sh  --issue  --staging  -d demo.domain --renew-hook "/root/.acme.sh/renew-hook-update-Apache APISIX.sh  -h http://Apache APISIX-admin:port -p /root/.acme.sh/demo.domain/demo.domain.cer -k /root/.acme.sh/demo.domain/demo.domain.key -a xxxxxxxxxxxxx"
```
Comment (tao12345666333) [inline docs/en/latest/FAQ.md:498]: ditto

File: docs/en/latest/FAQ.md [hunk 17]
```diff
@@ -476,26 +476,26 @@ The route was created successfully. It means that the modification of `X-API-KEY
...
+**Note: You can use this method in a non-production environment to allow all clients from anywhere to access your `Apache Apache APISIX` instances, but it is not safe to use it in a production environment. In production environment, please only authorize specific IP addresses or address ranges to access your instance.**
 
 ## How to auto renew SSL cert via acme.sh
 
 ```bash
-$ curl --output /root/.acme.sh/renew-hook-update-apisix.sh --silent https://gist.githubusercontent.com/anjia0532/9ebf8011322f43e3f5037bc2af3aeaa6/raw/65b359a4eed0ae990f9188c2afa22bacd8471652/renew-hook-update-apisix.sh
+$ curl --output /root/.acme.sh/renew-hook-update-Apache APISIX.sh --silent https://gist.githubusercontent.com/anjia0532/9ebf8011322f43e3f5037bc2af3aeaa6/raw/65b359a4eed0ae990f9188c2afa22bacd8471652/renew-hook-update-Apache APISIX.sh
 
-$ chmod +x /root/.acme.sh/renew-hook-update-apisix.sh
+$ chmod +x /root/.acme.sh/renew-hook-update-Apache APISIX.sh
```
Comment (tao12345666333) [inline docs/en/latest/FAQ.md:496]: ditto

File: docs/en/latest/FAQ.md [hunk 18]
```diff
@@ -476,26 +476,26 @@ The route was created successfully. It means that the modification of `X-API-KEY
...
 
-**Note: You can use this method in a non-production environment to allow all clients from anywhere to access your `Apache APISIX` instances, but it is not safe to use it in a production environment. In production environment, please only authorize specific IP addresses or address ranges to access your instance.**
+**Note: You can use this method in a non-production environment to allow all clients from anywhere to access your `Apache Apache APISIX` instances, but it is not safe to use it in a production environment. In production environment, please only authorize specific IP addresses or address ranges to access your instance.**
 
 ## How to auto renew SSL cert via acme.sh
 
 ```bash
-$ curl --output /root/.acme.sh/renew-hook-update-apisix.sh --silent https://gist.githubusercontent.com/anjia0532/9ebf8011322f43e3f5037bc2af3aeaa6/raw/65b359a4eed0ae990f9188c2afa22bacd8471652/renew-hook-update-apisix.sh
+$ curl --output /root/.acme.sh/renew-hook-update-Apache APISIX.sh --silent https://gist.githubusercontent.com/anjia0532/9ebf8011322f43e3f5037bc2af3aeaa6/raw/65b359a4eed0ae990f9188c2afa22bacd8471652/renew-hook-update-Apache APISIX.sh
```
Comment (tao12345666333) [inline docs/en/latest/FAQ.md:494]: ditto

File: docs/en/latest/FAQ.md [hunk 19]
```diff
@@ -476,26 +476,26 @@ The route was created successfully. It means that the modification of `X-API-KEY
...
+Apache APISIX:
   allow_admin:
     - 0.0.0.0/0
 ```
 
-Restart or reload APISIX, all IPs can access the `Admin API`.
+Restart or reload Apache APISIX, all IPs can access the `Admin API`.
 
-**Note: You can use this method in a non-production environment to allow all clients from anywhere to access your `Apache APISIX` instances, but it is not safe to use it in a production environment. In production environment, please only authorize specific IP addresses or address ranges to access your instance.**
+**Note: You can use this method in a non-production environment to allow all clients from anywhere to access your `Apache Apache APISIX` instances, but it is not safe to use it in a production environment. In production environment, please only authorize specific IP addresses or address ranges to access your instance.**
```
Comment (tao12345666333) [inline docs/en/latest/FAQ.md:489]: duplicated

File: docs/en/latest/FAQ.md [hunk 20]
```diff
@@ -432,16 +432,16 @@ HTTP/1.1 200 OK
...
 Note: There are security risks in using the default API token. It is recommended to update it when deploying to a production environment.
 
 2. `X-API-KEY` can be modified.
 
-For example: make the following changes to the `apisix.admin_key.key` in the `conf/config.yaml` file and reload APISIX.
+For example: make the following changes to the `Apache APISIX.admin_key.key` in the `conf/config.yaml` file and reload Apache APISIX.
 
 ```yaml
-apisix:
+Apache APISIX:
```
Comment (tao12345666333) [inline docs/en/latest/FAQ.md:444]: ditto

File: docs/en/latest/FAQ.md [hunk 21]
```diff
@@ -476,26 +476,26 @@ The route was created successfully. It means that the modification of `X-API-KEY
 
 ## How to allow all IPs to access Admin API
 
-By default, Apache APISIX only allows the IP range of `127.0.0.0/24` to access the `Admin API`. If you want to allow all IP access, then you only need to add the following configuration in the `conf/config.yaml` configuration file.
+By default, Apache Apache APISIX only allows the IP range of `127.0.0.0/24` to access the `Admin API`. If you want to allow all IP access, then you only need to add the following configuration in the `conf/config.yaml` configuration file.
```
Comment (tao12345666333) [inline docs/en/latest/FAQ.md:479]: duplicated

File: docs/en/latest/FAQ.md [hunk 22]
```diff
@@ -432,16 +432,16 @@ HTTP/1.1 200 OK
 
 ## What is the `X-API-KEY` of Admin API? Can it be modified?
 
-1. The `X-API-KEY` of Admin API refers to the `apisix.admin_key.key` in the `config.yaml` file, and the default value is `edd1c9f034335f136f87ad84b625c8f1`. It is the access token of the Admin API.
+1. The `X-API-KEY` of Admin API refers to the `Apache APISIX.admin_key.key` in the `config.yaml` file, and the default value is `edd1c9f034335f136f87ad84b625c8f1`. It is the access token of the Admin API.
```
Comment (tao12345666333) [inline docs/en/latest/FAQ.md:435]: ditto

**Review by tao12345666333 — CHANGES_REQUESTED** [review]

File: docs/en/latest/FAQ.md [hunk 23]
```diff
@@ -495,7 +495,7 @@ $ curl --output /root/.acme.sh/renew-hook-update-apisix.sh --silent https://gist
 
 $ chmod +x /root/.acme.sh/renew-hook-update-apisix.sh
 
-$ acme.sh  --issue  --staging  -d demo.domain --renew-hook "/root/.acme.sh/renew-hook-update-apisix.sh  -h http://apisix-admin:port -p /root/.acme.sh/demo.domain/demo.domain.cer -k /root/.acme.sh/demo.domain/demo.domain.key -a xxxxxxxxxxxxx"
+$ acme.sh  --issue  --staging  -d demo.domain --renew-hook "/root/.acme.sh/renew-hook-update-apisix.sh  -h http://apisix-admin: -p /root/.acme.sh/demo.domain/demo.domain.cer -k /root/.acme.sh/demo.domain/demo.domain.key -a xxxxxxxxxxxxx"
```
Comment (tao12345666333) [inline docs/en/latest/FAQ.md:498]: No need to modify this line.

Comment (yzeng25) [reply docs/en/latest/FAQ.md]: fixed

**Review by tao12345666333 — APPROVED** [review]

**Review by spacewander — APPROVED** [review]

**Pull request merged**